docs/tutorial.md

```diff
@@ -0,0 +1,177 @@
+
+```python
+pip install graphkb
+```
+
+## Connecting to the API
+
+The first thing to do is setting up the connection to the API
+
+```python
+from graphkb import GraphKBConnection
+
+graphkb_conn = GraphKBConnection()
+```
+
+Next, use this to login
+
+```python
+graphkb_conn.login(username, password)
+```
+
+This will store the credentials passed on the connection object and re-login as required.
+
+## Variant Matches
+
+For this example we are going to try matching a protein change (`p.G12D`) on the gene (`KRAS`).
+
+```python
+from graphkb.match import match_positional_variant
+
+variant_name = 'KRAS:p.G12D'
+variant_matches = match_positional_variant(graphkb_conn, variant_name)
+
+for match in variant_matches:
+    print(variant_name, 'will match', match['displayName'])
+```
+
+From this step you should see something like this (actual content will vary depending on the
+instance of the GraphKB API/DB you are using)
+
+```text
+KRAS:p.G12D will match KRAS:p.G12
+KRAS:p.G12D will match KRAS:p.G12X
+KRAS:p.G12D will match KRAS:p.G12D
+KRAS:p.G12D will match KRAS:p.G12mut
+KRAS:p.G12D will match KRAS:p.(G12_G13)mut
+KRAS:p.G12D will match KRAS:p.?12mut
+KRAS:p.G12D will match KRAS:p.G12D
+KRAS:p.G12D will match chr12:g.25398284C>T
+KRAS:p.G12D will match KRAS:p.G12mut
+KRAS:p.G12D will match KRAS mutation
+```
+
+As you can see above the match function has pulled similar/equivalent variant representations which
+we will then use to match statements.
+
+Next, use these variant matches to find the related statements
+
+## Statement Annotations
+
+```python
+from graphkb.constants import BASE_RETURN_PROPERTIES, GENERIC_RETURN_PROPERTIES
+from graphkb.util import convert_to_rid_list
+
+# return properties should be customized to the users needs
+return_props = (
+    BASE_RETURN_PROPERTIES
+    + ['sourceId', 'source.name', 'source.displayName']
+    + [f'conditions.{p}' for p in GENERIC_RETURN_PROPERTIES]
+    + [f'subject.{p}' for p in GENERIC_RETURN_PROPERTIES]
+    + [f'evidence.{p}' for p in GENERIC_RETURN_PROPERTIES]
+    + [f'relevance.{p}' for p in GENERIC_RETURN_PROPERTIES]
+    + [f'evidenceLevel.{p}' for p in GENERIC_RETURN_PROPERTIES]
+)
+
+statements = graphkb_conn.query(
+    {
+        'target': 'Statement',
+        'filters': {'conditions': convert_to_rid_list(variant_matches), 'operator': 'CONTAINSANY'},
+        'returnProperties': return_props,
+    }
+)
+
+for statement in statements[:5]:
+    print(
+        statement['relevance']['displayName'],
+        statement['subject']['displayName'],
+        statement['source']['displayName'] if statement['source'] else '',
+    )
+```
+
+This should output lines similar to the following
+
+```text
+resistance gefitinib [C1855] CIViC
+likely pathogenic lung cancer [DOID:1324] DoCM
+```
+
+## Categorizing Statements
+
+Something we often what to do is to know if a statement is therapeutic, or prognostic, etc. The
```

Comment:
typo: "Something we often what to do is to know"

Reply:
fixed!